FIPS is only tested on RHEL (#581)
theforeman/foreman-documentation: guides/common/modules/ref_system-requirements.adoc

.FIPS Mode
You can install {ProductName} on a {RHEL} system that is operating in FIPS mode.
ifndef::satellite[]

> {RHEL} clones are not being actively tested in FIPS mode, we do recommend to use {RHEL}.

Reviewer: ```suggestion
{RHEL} clones are not being actively tested in FIPS mode. If you require FIPS, consider using {RHEL}.
```
